### DISCUSSION TITLE
XiaoZhi Desktop compatibility

### DISCUSSION THREAD

**Discussion by wangyaokun86** [rocksdanister/lively#1341] [Q&A]
Adding .exe program to the application rules is the default for this program to pause playing?Can I play it under a specific .exe?

**Comment by rocksdanister** [comment]
I don't understand the question?.. try giving a detailed example of what you are trying to do.

**Reply by wangyaokun86** [reply]
Thank you~...My English is not good. I have a desktop finishing software, but every time I click the desktop wallpaper to pause playing, I want to reduce the burden of the computer (other applications pause playing when full screen), but I don't want to use this rule on the desktop finishing software. In fact, I just completed this idea by keeping it running and manually setting many full screen applications to pause~

**Reply by rocksdanister** [reply]
What is Desktop finishing software?

**Reply by wangyaokun86** [reply]
sorry...it's desktop classification software

**Reply by rocksdanister** [reply]
What is the **Name** of the software?

**Reply by wangyaokun86** [reply]
XiaoZhi Desktop(https://xzdesktop.cqttech.com/）

**Reply by rocksdanister** [reply]
Software like that which modify desktop will not be compatible with Lively unless the developer reach out and we work together supporting it.

Lively is compatible with Rainmeter (opensource and free):
https://www.rainmeter.net/

**Reply by wangyaokun86** [reply]
Ok, thank u for your patient reply~
